src/amp/server/ampExperimentCache.ts

```diff
@@ -0,0 +1,40 @@
+import fetch from 'node-fetch';
+
+export type AmpExperiment = {
+    sticky?: boolean,
+    consentNotificationId?: string,
+    variants: {
+        [key: string]: number; // floating point number
+    }
+}
+
+export type AmpExperiments = {
+    [key: string]: AmpExperiment
+}
+
+export const ampExperimentsDataUrl = process.env.GU_ENV === 'PROD'
+    ? 'https://contributions.guardianapis.com/amp/experiments_data'
+    : 'https://contributions.code.dev-guardianapis.com/amp/experiments_data';
+
+let ampExperimentCache: AmpExperiments = {};
+export const getAmpExperimentCache = (): AmpExperiments => ampExperimentCache
+
+const fetchExperimentsData = (): Promise<void> => {
+    // eslint-disable-next-line no-console
+    console.log(`------------------ called fetchExperimentsData at ${Date.now()} -----------------`)
+
+    return fetch(ampExperimentsDataUrl)
+        .then(rawResponse => rawResponse.json())
+        .then(json => {
+            ampExperimentCache = json
+        })
+}
+
+const oneMinute = 60_000;
```

Comment:
🤯  never knew we could do that!!! 🎊

Reply:
Which thing @liywjl?

Reply:
using `_` within numbers in JS, never knew you could do that 😃

Reply:
Oh right, yeah, it’s great for readability! 🙂